app/src/main/java/fi/bitrite/android/ws/ui/FavoriteUsersFragment.java

```diff
@@ -40,7 +39,7 @@
     @Inject UserRepository mUserRepository;
 
     @BindView(R.id.favorites_lst_users) ListView mLstUsers;
-    @BindView(R.id.favorites_lbl_no_users) TextView mLblNoUsers;
+    @BindView(R.id.favorites_lbl_no_users) View mLblNoUsers;
```

Comment:
It’s not better—it’s _correct_. `R.id.favorites_lbl_no_users` is not an instance of TextView in `fragment_favorite_users.xml`, but a RelativeLayout. But since the class doesn’t call any RelativeLayout-specific methods on the variable, `View` should suffice.

Reply:
Uh, I did not check that. Sorry. :)
Could you then please change its id to favorites_rellayout_no_users, thank you.